.gitignore

```diff
@@ -3,4 +3,6 @@ dist
 coverage
 .DS_Store
 .vscode
-src/**.js
```

Comment:
Might be nice to just throw some base exclusions in here for various IDE's/project specific things
example: https://gist.github.com/chhh/4961200